src/scr.c

```diff
@@ -678,19 +679,37 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      // 1. Dump SCR parameters - SCR_ENABLE 
+      scr_dbg(0, "SCR_ENABLE is disabled");
+    }
     return SCR_FAILURE;
   }
-
+  else {
+    if (scr_my_rank_world == 0) {
+      // 2. Dump SCR parameters - SCR_ENABLE
+      scr_dbg(0, "SCR_ENABLE is enabled");
+    }
+  }
   /* set debug verbosity level */
   if ((value = scr_param_get("SCR_DEBUG")) != NULL) {
     scr_debug = atoi(value);
+    if (scr_my_rank_world == 0) {
+      // 3. Dump SCR parameters - SCR_aDEBUG
+      scr_dbg(scr_debug, "SCR_DEBUG level = %d", scr_debug);
+    }
   }
 
   /* set scr_prefix_path and scr_prefix */
   value = scr_param_get("SCR_PREFIX");
   scr_prefix_path = scr_get_prefix(value);
   scr_prefix = spath_strdup(scr_prefix_path);
 
+  if (scr_my_rank_world == 0) {
+      // 4. Dump SCR parameters - SCR_PREFIX
+      scr_dbg(scr_debug, "SCR_PREFIX = %s ", scr_prefix);
+  }
+
```

Comment:
```suggestion
  if (scr_my_rank_world == 0) {
      // 4. Dump SCR parameters - SCR_PREFIX
      scr_dbg(scr_debug, "SCR_PREFIX=%s", scr_prefix);
  }

```

Reply:
I like the spaces in `` = `` for readability.  However, I think it will be more useful to drop the spaces.  That way it's easier from someone to cut-and-paste this output and use those lines to set environment variables in a bash shell.  I keep going back on forth on it, but let's try to remove the spaces everywhere for now and see how that looks.